src/common/templates/_storage.tpl

```diff
@@ -6,6 +6,12 @@
         {{- $storageClass = .global.storageClass -}}
     {{- end -}}
 {{- end -}}
+{{/*
+local values have more precedence over global values.
+*/}}
+{{- if .persistence.storageClass -}}
```

Comment:
Could we check if "persistence" field is present before accessing "storageClass"?